### PR TITLE
86b1u14wh

### PR DESCRIPTION
**Pull request by AynulAmna**
Changes has been done as per the review . Now, mentor able to register students and it will be stored in DB.

### CODE REVIEW

File: server/src/services/prisma.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { PrismaClient } from '@prisma/client';
```
Comment (bsaranga) [inline server/src/services/prisma.ts]: Let's use the Prisma singleton once Lakshanan commits it

File: server/src/app/mentor-register/page.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        firstName,
+        lastName,
+        email,
+        passwordHash: hashedPassword,
+        role: 'student', 
+      }
+    });
+
+     // Redirect to the success page after registration - demo page for now
+     redirect('/mentor-register/success');
```
Comment (bsaranga) [inline server/src/app/mentor-register/page.tsx]: Let's not redirect. Making this a client component and using a server action is much preferred

**Review by bsaranga — CHANGES_REQUESTED** [review]

File: server/src/services/prisma.ts (hunk 1 above)
Comment (AynulAmna) [reply server/src/services/prisma.ts]: done . please check and review

File: server/src/app/mentor-register/page.tsx (hunk 2 above)
Comment (AynulAmna) [reply server/src/app/mentor-register/page.tsx]: used server action please review

**Comment by AynulAmna** [comment]
Conflicts resolved

**Review by bsaranga — APPROVED** [review]
The random password generated must be picked up by the emailing module and sent to student email. @AynulAmna connect with the person who made the emailing module.

**Pull request merged**